include/deployer.bash

```diff
@@ -275,7 +275,10 @@ init-profile() {
     if [ -f $CBD_PROFILE ]; then
         debug "Use existing profile: $CBD_PROFILE"
         if [[ "$CBD_PROFILE" != *\/* ]]; then
-          debug "$CBD_PROFILE file will be searched in your $PATH not just the current directory"
+          PROFILES_ON_PATH="$(find $(echo $PATH | sed 's/:/ /g') -name 'Profile' -maxdepth 1 2>/dev/null | sort -u)" || true # for some reason I have to or true this
```

Comment:
I have concerns about not quoting $PATH. If a directory in the path has space looks like it is treated as two separate directories. I am suspecting that is causing find to return 1 because of which you have to "or true" it?